lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -82,6 +82,8 @@ defmodule LivebookWeb.SessionLive.CellComponent do
         <.move_cell_up_button cell_id={@cell_view.id} />
         <.move_cell_down_button cell_id={@cell_view.id} />
         <.delete_cell_button cell_id={@cell_view.id} />
+        <.remove_output_from_output_panel_button cell_id={@cell_view.id} />
+        <.add_output_to_output_panel_button cell_id={@cell_view.id} />
```

Comment:
I also need to save the state if a cells output is already added to the output panel.
Should I put it in %Notebook.Cell{output_location: :notebook | :output_panel}?

Reply:
The information is already in `notebook.outputs_panel`, so we can infer it from there.

Reply:
implemented in https://github.com/livebook-dev/livebook/pull/2109/commits/53038a5c119cbbdc0265bdc1909aa1a54f2072cf